tools/minicluster/minicluster.go

```diff
@@ -384,7 +384,7 @@ func main() {
 				return
 			}
 
-			log.S().Info("Fp token transfer test pass!")
+			log.S().Debug("Fp token transfer test pass!")
```

Comment:
revert. minicluster is not running in product

Reply:
fixed.